src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/memories/JqlMemory.kt

```diff
@@ -3,5 +3,10 @@ package com.atlassian.performance.tools.jiraactions.api.memories
 import com.atlassian.performance.tools.jiraactions.api.page.IssuePage
 
 interface JqlMemory : Memory<String> {
+
     fun observe(issuePage: IssuePage)
+
+    fun recallByTag(filter: (String) -> Boolean): String? {
```

Comment:
This requires `kotlin.jvm.functions.Function1`.
We can make it more portable if we switch to `java.util.function.Predicate`.

Reply:
It is done the same fashion as IssueMemory recall is.

Reply:
@drspartez IssueMemory is already part of an API. It's hard to fix it. I see no point in repeating the mistake here.

Reply:
Ok. I didn't know t is mistake.